source/includes/_node_rpc.md

```diff
@@ -0,0 +1,51 @@
+# Node RPC Calls
+
+```shell--curl
+curl $url/ \
+  -H 'Content-Type: application/json' \
+  -X POST \
+  --data '{ "method": "methodname", "params": [...] "id": "some-id" }'
+```
+
+```shell--cli
+bcoin cli rpc params...
+```
+
+```javascript
+const rpc = new bcoin.http.RPCClient({
+  network: 'testnet'
+});
+
+(async () => {
+  const res = await rpc.execute('MethodName', [ ...params ]);
+
+  // RES will return "result" part of the object, not the id or error
+  // error will be thrown.
+})().catch((err) => {
+  console.error(err.stack);
+});
+```
+
+> The above command returns JSON structured like this:
+
+> bcoin cli rpc and javascript will return error or result.
```

Comment:
I think these other two comments can be moved to the normal text section, since they're really just notes.